UserInterface.adoc

```diff
@@ -80,52 +128,368 @@ fundamental A: If it does behave that way now, I'd think it would be wise to
+
+The scope of that is, understandably, very large, so it warrants
+<<_main_panel, its own section in this part of the manual, visible below>>.
+
+==== Virtual keyboard panel
+
+The virtual keyboard is located at the bottom of the Zyn-Fusion interface.
+
+// TODO
+
+Modulation wheel::
+    Desc
+
+Keyboard::
+    Desc
+
+VELOCITY::
+    Desc
+
+VRND::
+    Desc
+
+OCTAVE::
+    Desc
+
+C.VAL::
+    Desc
+
+MIDI CC dropdown menu::
+    Desc
 
 === Main Panel
 
-Here's where the most of the action happens.
-When you start Zyn-Fusion it'll begin by showing you the Browser.
-Use the Side Bar to select what should be displayed here.
+The main panel covers all the core functionality in Zyn-Fusion,
+in regard to sound design, effects, mixing, and so on.
+
+We'll go over each subpanel from top to bottom as seen in the <<_navigation_panel, navigation panel>>.
+
+==== Part settings
+
+// TODO: Image with colors, or multiple images
+
+This subpanel is dedicated to setting up each the properties of each part.
+
+.What is a part?
+NOTE: A part is a single element in ZynAddSubFX capable of generating sound.
+In one part, any of ZynAddSubFX's three synths can be active.
+So, think of parts like mini synths inside ZynAddSubFX
+which you can use to have multiple layers to sounds,
+have multiple sounds, maybe with different timbres, on one keyboard, etc.
+
+===== Part enabling and naming
+
+This subpanel consists of pairs of a button and a text input box.
+The button enables the appropriate part, and the input box allows naming the part.
+
+===== Instrument settings
+
+VOLUME:: Volume, expressed in dB.
+PAN:: Panning.
+MIN KEY:: The minimum key from which the part starts playing
+MAX KEY:: The maximum key to which the part plays
+
+// TODO
+VEL SENSE::
+    Velocity sensing.
+
+VEL OFFSET::
+    Velocity offset.
+
+KEY SHIFT::
+    Shifts the key being played, same as KEY SHIFT in the <<_status_panel, Status panel>>.
+
+Mn/R/Mx buttons::
+    These buttons are used to set the minimum and maximum key that can be played.
+    `Mn` sets the minimum key to the key last given to Zyn-Fusion as input,
+    `Mx` does the same, except for the maximum key
+    and `R` resets the limits back to cover the full keyboard.
+
+Active MIDI channel::
+    Shows which MIDI channel the part will except input from.
+    By default, this is set to the ID of the part, i.e. CH1 for part 1, CH2 for part 2, etc.
+    One part can only except input from one MIDI channel.
+    In the case of music made using a DAW, you'll usually do everything on channel 1.
+    In the case of live music, you may use more channels, across different parts of the keyboard.
+
+Polyphony type::
+    *POLY* means that each input will be added to the output as a separate voice.
+    Use this if at any point you have two notes playing at the same time.
+    *MONO* means only one key will be accepted and played back. If a new note is
+    being played while another one is already on, the new one will playback.
```

Comment:
Perhaps to add to this (and to de-emphasize other things), talk about _why_ you might want one mode over another in terms of composition and historically how the options fit in.